### PR TITLE
Meine Republik Tweaks

### PR DESCRIPTION
**Pull request by ovbm**
- Keine Callouts in "Meine Republik" Actionbar
- Filter: "Gemerkt" statt "Lesezeichen"
- MiniFeed: Weiterlesen statt Lesezeichen

### CODE REVIEW

File: components/Bookmarks/queries.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ export const getCollectionItems = gql`
         after: $cursor
         progress: $progress
         uniqueDocuments: true
+        lastDays: 30
```
Comment (tpreusse) [inline components/Bookmarks/queries.js:33]: should be a null-able variables and only set to 30 when in «weiterlesen» view

File: components/Bookmarks/BookmarkMiniFeed.js [hunk 2]
```diff
@@ -10,7 +10,8 @@ import Link from '../Link/Path'
 import { getCollectionItems, registerQueryVariables } from './queries'
 
 const variables = {
-  collections: ['bookmarks']
+  collections: ['progress', 'bookmarks'],
+  progress: 'UNFINISHED'
```
Comment (tpreusse) [inline components/Bookmarks/BookmarkMiniFeed.js:14]: should be moved into a wrapper around the whole thing which does a different query based on `progressConsent`:
```
  const hasProgress = !!(me?.progressConsent)
  const variables = useMemo(() => {
    if (hasProgress) {
        return {
          collections: ['progress', 'bookmarks'],
          progress: 'UNFINISHED',
          lastDays: 30
        }
    }
    return {
      collections: ['bookmarks']
    }
  }, [hasProgress])
  registerQueryVariables(variables)
  return <BookmarkMiniFeed variables={variables} />
```

And then use a [options function](https://www.apollographql.com/docs/react/api/react/hoc/#configoptions) in the `graphql` wrapper:
```
  graphql(getCollectionItems, {
    options: props => ({
      fetchPolicy: 'cache-and-network',
      variables: props.variables
    })
  })
```

File: components/Bookmarks/queries.js (hunk 1 above)
Comment (ovbm) [reply components/Bookmarks/queries.js:33]: ah ja logo XD

**Review by tpreusse — APPROVED** [review]

File: components/Front/index.js [hunk 3]
```diff
@@ -121,18 +120,15 @@ const Front = ({
...
-    [isTester]
+  const schema = useMemo(() =>
+    createFrontSchema({
+      Link: HrefLink,
+      CommentLink,
+      DiscussionLink,
+      ...withData,
+      ActionBar,
+      t
+    })
```
Comment (tpreusse) [inline components/Front/index.js:131]: Watch out: we need an empty array here:
> If no array is provided, a new value will be computed on every render.
> [useMemo docs](https://reactjs.org/docs/hooks-reference.html#usememo)

**Pull request merged**